server/proxy/bindings.go

```diff
@@ -87,20 +92,19 @@ func (p *Proxy) GetBindingsForApp(sessionID, actingUserID string, cc *apps.Conte
 
 	resp := p.Call(sessionID, actingUserID, bindingsRequest)
 	if resp == nil || resp.Type != apps.CallResponseTypeOK {
-		// TODO Log error (chance to flood the logs)
-		// p.mm.Log.Debug("Response is nil or unexpected type.")
-		// if resp != nil && resp.Type == apps.CallResponseTypeError {
-		// 	p.mm.Log.Debug("Error getting bindings. Error: " + resp.Error())
-		// }
+		if resp != nil && resp.Type == apps.CallResponseTypeError {
```

Comment:
Could we combine the two `if`s in one? A bit counter-intuitive to have `resp == nil` at the top and then `resp != nil` inside.

Reply:
Not really. The best flow that I found is:
```go
resp := p.Call(sessionID, actingUserID, bindingsRequest)
if resp == nil || (resp.Type != apps.CallResponseTypeError && resp.Type != apps.CallResponseTypeOK) {
	logger.Debugf("Bindings response is nil or unexpected type.")
	return nil, nil
}

if resp.Type == apps.CallResponseTypeError {
	logger.Debugf("Error getting bindings. Error: " + resp.Error())
	return nil, nil
}
```

@catalintomai Let me know what you think

Reply:
fwiw I like it better. 1/5.

Reply:
Changed in 4c5a3b62bc866a589a3d894950a4c67b88c25b66